packages/ui/src/components/StoryCard/StoryCard.tsx

```diff
@@ -0,0 +1,106 @@
+import type { Culture, UploadedImage } from '@prezly/sdk';
+import Link from 'next/link';
+import { twMerge } from 'tailwind-merge';
+
+import type { StoryWithImage } from '@/types';
+
+import { CategoriesList } from '../CategoriesList';
+import { StoryPublicationDate } from '../StoryPublicationDate';
+
+import { StoryImage, type StoryImageSize } from './components';
+
+export interface Props {
+    story: StoryWithImage;
+    locale: Culture['code'];
+    size?: StoryImageSize;
+    newsroomName: string;
+    logo?: UploadedImage | null;
+    showDate?: boolean;
+    hideSubtitle?: boolean;
+    className?: string;
+}
+
+export function StoryCard({
+    story,
+    showDate,
+    locale,
+    newsroomName,
+    logo,
+    size = 'large',
+    hideSubtitle,
+    className,
+}: Props) {
+    const { slug, categories, title, subtitle } = story;
+
+    return (
+        <div
+            className={twMerge(
+                'flex flex-row items-center p-0 bg-white group sm:max-w-[530px]',
+                size === 'large' ? `flex-col gap-0` : `md:flex-row items-start gap-4`,
+                className,
+            )}
+        >
+            <Link href={`/${slug}`} locale={false} legacyBehavior>
+                <div
+                    className={twMerge(
+                        size === 'large'
+                            ? 'w-full aspect-[20/14] md:aspect-[27/17]'
+                            : 'w-1/3 aspect-[10/7] hidden md:block',
```

Comment:
Isn't `10/7` the same as `20/14`?

Reply:
Is the same, but the large has a different aspect ration from tablet to desktop viewport

Reply:
I'm just wondering why the same aspect ratio is used with two different notations. `20/14` can be replaced with `10/7` with no side effects. Right?

Reply:
Yeah right

Reply:
Updated https://github.com/prezly/theme-kit-js/pull/414/commits/7f5e4071f2ebd39b7cae32883fec555728b1ac8b